pages/pil/update/index.js

```diff
@@ -29,7 +29,13 @@ module.exports = settings => {
       establishment: req.establishment.id
     };
     req.user.can('pil.update', params)
-      .then(can => can ? next() : next(new Error('Unauthorised')))
+      .then(can => {
+        const authorised = can && req.pil.establishmentId === req.establishment.id;
```

Comment:
I'd probably expect this check to be incorporated into the permissions API, which to my understanding should till check that a user is either the PIL holder or an admin at the holding establishment.

I believe this can be done with the `params` change above.